.env.example

```
@@ -58,8 +58,8 @@ LOG_LEVEL=info
 
 EVENTBRITE_OAUTH2_PERSONAL_TOKEN=
 
-RECAPTCHA_PUBLIC_KEY=
-RECAPTCHA_PRIVATE_KEY=
+TURNSTILE_SITE_KEY=
+TURNSTILE_SECRET_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TURNSTILE_SECRET_KEY key should go before the TURNSTILE_SITE_KEY key

(UnorderedKey)